src/main/java/com/epam/grid/engine/provider/job/slurm/SlurmJobProvider.java

```diff
@@ -263,6 +318,42 @@ private Listing<Job> mapToJobListing(final List<String> stdOut) {
     }
 
     private boolean jobNotFoundByIdError(final CommandResult result) {
-        return result.getStdErr().toString().equals(jobIdNotFoundMessage);
+        if (!result.getStdErr().isEmpty()) {
```

Comment:
try to rewrite this with `streams`, and also please check message not only for first string but all

Reply:
could you please fix line `149` with the same manned as here instead of `toString()`  collect all line from `stdErr`

Reply:
done